app/index.tsx

```diff
@@ -6,7 +6,7 @@ import { useShallow } from "zustand/react/shallow"
 
 import { ProviderContextProvider } from "~app/context/provider"
 import { TransactionAuthorization } from "~app/page/authorization/transaction"
-import { Info } from "~app/page/info"
+import { Home } from "~app/page/info"
```

Comment:
nit: Also rename the directory to `home`
```suggestion
import { Home } from "~app/page/home"
```